### DISCUSSION TITLE
Faulty Scaling of PSE @ 3440x1440p

### DISCUSSION THREAD

**Discussion by ghost** [rocksdanister/lively#1268] [General]
**Describe the bug**
The included PSE wallpaper sadly doesnt scale correctly @ 3440x1440p resolution.
Both the lanthanide and actinide row are below the taskbar.

**To Reproduce**
1. Select the included Periodic Table wallpaper
2. Have a ultrawide 3440x1440p resolution
3. See, that you dont see the lanthanides and actinides

**Expected behavior**
To be able to see the lanthanide and actinide rows, as the wallpaper should be refitted correctly to fit the ultrawide 3440x1440p resolution (as in the preview window).

**Screenshots/Video**
See screenshot for detailed view of the faulty rescaling - chopping off the lanthanides and actinides.
![PSE_scaling](https://user-images.githubusercontent.com/107806588/174492511-fab51ed2-0582-4988-a06e-3ae756bb8ed4.jpg)

**Desktop**
- OS Edition: Windows 10 Pro; OS Version: 21H2; OS Build: 19044.1766
- Wallpaper running: Periodic Table

**Log file**
[lively_log_20220619_191131.zip](https://github.com/rocksdanister/lively/files/8936013/lively_log_20220619_191131.zip)

**Comment by dabjulmaros** [comment]
[periodic_table.zip](https://github.com/rocksdanister/lively/files/8985356/periodic_table.zip)
I have modified this version of the wallapaper to be user adjustable.
Once you import it you should see two controls, one for the table width, and one for the text size multiplier.
You should be able to adjust the width to make sure the whole table is displayed, then adjust the text size to avoid overflow, or the text being too small.

**Reply by crLmsin** [reply]
Any possible help on this? I try to import this, and it says 'Error: Not Lively Wallpaper file'. can you help?

**Reply by dabjulmaros** [reply]
try this one
[periodic_table.zip](https://github.com/rocksdanister/lively/files/14018372/periodic_table.zip)

**Reply by crLmsin** [reply]
That worked. Thank you!
